clients/gtest/gemm_strided_batched_gtest.yaml

```diff
@@ -975,7 +977,7 @@ Tests:
     - gemm_strided_batched_ex: *nonint8_real_precisions
     - gemm_strided_batched_ex: *int8_precision
   arguments:
-    - { M: 4, N: 4, K: 4, lda: 4, ldb: 4, ldc: 4, stride_a: 4, stride_b: 4, stride_c: 4, stride_d: 4 }
```

Comment:
I think you'll have to change rocblas_gentest.py to include gemm_strided_batched for these to be calculated correctly. A quick search didn't find that function included there.

Reply:
> I think you'll have to change rocblas_gentest.py to include gemm_strided_batched for these to be calculated correctly. A quick search didn't find that function included there.

Okay now I recall we didn't want to slow down gemms with stride_scale system, probably should abort when not processed.  Tests were running so will debug what value got used....